osu.Game/Overlays/Music/Playlist.cs

```diff
@@ -75,10 +71,10 @@ public void Filter(string text)
             {
                 var metadata = playlistItem.BeatmapSetInfo.Metadata;
                 var match = string.IsNullOrEmpty(text)
-                    || (metadata.Artist ?? "").IndexOf(text, StringComparison.InvariantCultureIgnoreCase) != -1
-                    || (metadata.ArtistUnicode ?? "").IndexOf(text, StringComparison.InvariantCultureIgnoreCase) != -1
-                    || (metadata.Title ?? "").IndexOf(text, StringComparison.InvariantCultureIgnoreCase) != -1
-                    || (metadata.TitleUnicode ?? "").IndexOf(text, StringComparison.InvariantCultureIgnoreCase) != -1;
+                            || (metadata.Artist ?? "").IndexOf(text, StringComparison.InvariantCultureIgnoreCase) != -1
+                            || (metadata.ArtistUnicode ?? "").IndexOf(text, StringComparison.InvariantCultureIgnoreCase) != -1
+                            || (metadata.Title ?? "").IndexOf(text, StringComparison.InvariantCultureIgnoreCase) != -1
+                            || (metadata.TitleUnicode ?? "").IndexOf(text, StringComparison.InvariantCultureIgnoreCase) != -1;

```

Comment:
It may sound stupid, but I'm not sure where to extract such code in osu! codebase.
Is it ok to create static helper classes?
For this case, `FilteringHelper` with `Filter(IEnumerable<BeatmapSetInfo> beatmapSets, string query)`.